app/components/alert_icon_component.rb

```diff
@@ -0,0 +1,37 @@
+class AlertIconComponent < BaseComponent
+  ICON_SOURCE = {
+    warning: 'status/warning.svg',
+    error: 'status/error.svg',
+    error_lock: 'status/error-lock.svg',
+    personal_key: 'personal-key/personal-key.svg',
```

Comment:
We can move it there if you feel strongly about it. There was already an `app/assets/images/personal-key/` directory with some assets in it, and previously the `p-key.svg` (which this replaces) was up a level in `app/assets/images/`. Whatever we decide, I don't see a need to keep two copies around in case this gets used elsewhere

Reply:
I think if the expectation is that there'd a pretty tight coupling between the image and this component (seems fair?), it would make sense to have all of those component images together.